build/services/calcService.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.monthlyPremium = exports.yearlyPremium = void 0;
+let yearly;
+let monthly;
+const yearlyPremium = (carValue, riskRating) => {
+    //This if statement makes sure that invalid numbers cannot be submitted
+    if (!(carValue > 0 && riskRating > 0)) {
+        throw new Error("Car value and risk rating must be greater than 0");
+    }
+    yearly = (carValue * riskRating) / 100;
+    return yearly;
+};
+exports.yearlyPremium = yearlyPremium;
```

Comment:
Noticed in calcController.js your calcPremiums function has a catch block to handle errors. Do you think you can apply a catch block on this yearlyPremium function too? Below is a suggestion but I'm sure you can come up with cleaner code.
```suggestion
const yearlyPremium = (carValue, riskRating) => {
  try {
    if (!(carValue > 0 && riskRating > 0)) {
      throw new Error("Car value and risk rating must be greater than 0");
    }
    yearly = (carValue * riskRating) / 100;
    return yearly;
  } catch (error) {
    // handle the error here
    console.error(error);
  }
};

```